conf/machine/orange-pi-zero2.conf

```diff
@@ -0,0 +1,19 @@
+#@TYPE: Machine
+#@NAME: orange-pi-zero-2
+#@DESCRIPTION: Machine configuration for the orange-pi-zero-2, based on Allwinner H616 CPU
+
+require conf/machine/include/sun50i.inc
+
+PREFERRED_PROVIDER_virtual/kernel = "linux-mainline"
+
+KERNEL_DEVICETREE = "allwinner/sun50i-h616-orangepi-zero2.dtb"
+UBOOT_MACHINE = "orangepi_zero2_defconfig"
+
+UBOOT_BINARY = "u-boot-nodtb.bin"
+SPL_BINARY = "u-boot-sunxi-with-spl.bin"
```

Comment:
Those lines are also not necessary

Reply:
Hmm, I think that SPL_BINARY value being set here is actually needed - when I comment this out, then the board does not boot. 
I see that in sunxi.inc (not used by orange-pi-zero2.conf) this variable is set to the same value as here, however, in the sunx64 (which is included by sun50i.inc and then by orange-pi-zero2.conf) the SPL_BINARY variable has a different value. Why is that?

Reply:
Don't know why is that set in sunxi64. Then keep it set please.